common/blocks.py

```diff
@@ -438,12 +438,12 @@ class Meta:
 
 
 class SimpleIncidentSet(blocks.StructBlock):
-    category = blocks.PageChooserBlock(
+    category = blocks.ListBlock(blocks.PageChooserBlock(
```

Comment:
It's looking to me like if we change this to a list block, pages that have a single value here are throwing an error in the wagtail admin (for me it's `TypeError: object of type 'NoneType' has no len()`). Though can you verify this? My local DB might be in a weird state. It's happening to me on pages with a vertical bar chart from before this branch was created.

If it is the case that it's causing an error, I could see two options: 1. we write a data migration to update the existing pages to have a list rather than a single value for the any blocks that use Simple Incident Set. I don't think it's a trivial problem but it probably can be done. Option 2: add a new field `categories` alongside this one that exposes the new behavior, migrate the old charts to the new format, then remove `category` once we're sure that's happened. This one has more steps but is probably safer.

Reply:
Yeah, we'd have to create a migration ourselves manually to, like, loop through all the blog pages and their blocks and if any of them has a simple incident set, update the category to be either `[]` or `[value]` if there's a value. Conceptually it's straightforward but, the last time I looked, anyway, the API for manipulating `StreamField` values, especially nested ones like we have here, is _not_ the most straightforward or user-friendly. I guess [here is the documentation](https://docs.wagtail.org/en/v4.1.5/topics/streamfield.html#modifying-streamfield-data). 

But the fact that we probably don't have that many pages using these charts yet combined with my general overall wariness about automated modifications to pages (which in this case would potentially affect every blog page) is making me lean towards the two-phase approach. Plus this gives us a chance to make the name plural, and also add some help text about what effect choosing multiple categories has (i.e. does it filter for _all_ choices or _any_ choice)). So maybe let's go with that one.